src/txmempool.cpp

```diff
@@ -752,8 +752,9 @@ void CTxMemPool::removeConflicts(const CTransaction &tx, std::list <CTransaction
         if (it != mapNextTx.end()) {
             const CTransaction &txConflict = *it->second;
             if (txConflict != tx) {
+                uint256 const txConflictHash = txConflict.GetHash();
```

Comment:
Better to change to `uint256 const &txConflictHash` or revert it back to prevent copy constructor.

Reply:
Nope, you cannot do the reference here. Also isn't this a bug in the bitcoin code?

Reply:
What is the problems? `txConflict.GetHash()` already return `const uint256&`.

Reply:
@a-bezrukov just explain what is going on here. I think will be better to put a comment here why we need to copy to prevent somebody accidentally change it in the future.